src/faq.md

```diff
@@ -75,40 +75,6 @@ about the VM question in more detail.
 
 ## Language
 
-### Q. Isn't Dart a lot like Java?
```

Comment:
Let's add a TODO to explain the current differences from Java. It's still a valid question to ask, and there definitely are differences — if nothing else, the terseness and the more developer-friendly type system.